src/Rules/Properties/PropertiesInInterfaceRule.php

```diff
@@ -66,6 +66,15 @@ public function processNode(Node $node, Scope $scope): array
 			];
 		}
 
+		if ($node->isStatic()) {
+			return [
+				RuleErrorBuilder::message('Interfaces cannot include static hooked properties.')
+					->nonIgnorable()
+					->identifier('property.staticInInterface')
```

Comment:
I think we should use the exact same message and identifier as in PropertyInClassRule.